package.json

```diff
@@ -23,15 +23,18 @@
     "network",
     "browser"
   ],
-  "main": "./dist/vis-network.min.js",
```

Comment:
I don't know. It seems pretty straight forward to me that `node_modules/vis-network/dist/umd.min.js` is the UMD build of vis-network package.

And why would it be a major change? You import the package, the package.json has main that points to a file, the path of the file doesn't matter.

Reply:
Also we have it this way in vis-util, vis-uuid and vis-data. If it's too cryptic and a major change we should discuss it there too.

Reply:
Oh, it breaks everything **inside** of this package :smile:.

Reply:
The problem is, that not everybody used a package manager. In a lot if use-cases people still just integrate a script tag with the library they need. So if the file `dist/vis-network.min.js` suddenly does't exist anymore than this is breaking change.

I have no problem with this technical name in the `vis-uuid`, `vis-util` and even `vis-data` package because thay are basically just helper packages that are propably never integrated in a website.

I still would ask you to keep the name (at least for now).